hack/init_postgres.sql

```diff
@@ -72,6 +72,24 @@ CREATE TABLE auth.audit_log_entries (
 CREATE INDEX audit_logs_instance_id_idx ON auth.audit_log_entries USING btree (instance_id);
 comment on table auth.audit_log_entries is 'Auth: Audit trail for user actions.';
 
+-- auth.nonces definition
+CREATE TABLE auth.nonces (
+	instance_id uuid NULL,
+	id uuid NOT NULL,
+	
+	hashed_ip varchar(255) NOT NULL,
+
+	uri text NOT NULL,
+	wallet_address text NOT NULL,
+	chain_id integer NOT NULL,
+
+	created_at timestamptz NULL,
+	expires_at timestamptz NULL,
+	CONSTRAINT nonces_pkey PRIMARY KEY (id)
+);
+CREATE INDEX nonces_instance_id_idx ON auth.nonces USING btree (instance_id);
+comment on table auth.nonces is 'Auth: Stored generated nonces used for validating Web3 login requests.';
+
```

Comment:
The `init_postgres.sql` file is used to initialise the postgres database for local development. Gotrue runs the migrations on start so any updates to the database (new tables / schemas, etc) should be added as a new migration.

Reply:
All sorted now!